specification/communication/data-plane/Messages/stable/2024-02-01/examples/NotificationMessages_Send.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Sends a notification message",
+  "operationId": "NotificationMessagesOperations_Send",
+  "parameters": {
+    "api-version": "2024-02-01",
+    "content-type": "application/json",
+    "body": {
+      "channelRegistrationId": "0f0eb78e-a576-1dfc-b414-0f0756de3338",
+      "to": [
+        "14250000000"
+      ],
+      "type": "text",
```

Comment:
Should this be `kind`?

Reply:
Ah, this is generated, same comments as under Communication.Messages :-)

Reply:
updated and regenerated